crates/api/src/comment/like.rs

```diff
@@ -18,13 +18,12 @@ use lemmy_db_schema::{
 };
 use lemmy_db_views::structs::{CommentView, LocalUserView};
 use lemmy_utils::error::{LemmyError, LemmyErrorExt, LemmyErrorType};
-use std::ops::Deref;
 
 #[tracing::instrument(skip(context))]
 pub async fn like_comment(
   data: Json<CreateCommentLike>,
   context: Data<LemmyContext>,
-) -> Result<Json<CommentResponse>, LemmyError> {
+) -> Result<Json<SuccessResponse>, LemmyError> {
```

Comment:
Some of these, like this one, shouldn't return a simple success IMO. Remember that liking a comment, say 2 minutes after viewing it, might mean that it's score jumps up more than 1, because other people have liked it.